src/pages/NotFound/index.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+
+export default () => {
+  const history = useHistory();
+
+  return (
+    <p role="alert">
+      Sorry, we can't find the page {history.location.pathname}!
+    </p>
+  );
```

Comment:
delete if not using?

Reply:
We are using technically. If someone chenges the url directly then it will show them this page for anything other than "/"